cmd/helm-sync/main.go

```diff
@@ -38,9 +38,9 @@ var (
 	flVersion = flag.String("version", os.Getenv(reconcilermanager.HelmChartVersion),
 		"the version of the helm chart being synced")
 	flValues = flag.String("values", os.Getenv(reconcilermanager.HelmValues),
-		"set the helm chart values, will be used to override the default values")
-	flValuesFileSources = flag.String("values-file-sources", os.Getenv(reconcilermanager.HelmValuesFileSources),
-		"set the helm chart values, will be used to override the default values")
+		"inline helm chart values, will be used to override the default values")
```

Comment:
what's the syntax here? json? key=value comma delimited?

Reply:
added to the description